tools/eslint-rules/align-function-arguments.js

```diff
@@ -32,8 +32,7 @@ function checkArgumentAlignment(context, node) {
     'ArrowFunctionExpression',
     'CallExpression',
     'FunctionExpression',
-    'ObjectExpression',
-    'TemplateLiteral'
+    'ObjectExpression'
```

Comment:
I'm a supporter of the final-line dangling-comma but don't see it too much in the code base here. Happy to add an instance! And...

...done!